docs/note.md

```diff
@@ -266,7 +265,71 @@ Instead use a video div with two sources and a poster that gets loaded as a prev
 
 {{<video webm_src="../img/heart.webm" mp4_src="../img/heart.mp4" alt="A robot drawing a heart" poster="../img/heart.jpg">}}
 
-or if you want a video without controls - mimicking a GIF:
+We use `webm` and `mp4` source files for videos because they are generally smaller.
+The poster is an image that gets loaded as a preview.
+
+To create the `webm` and mp4 files use these commands:
+
+{{< tabs >}}
+{{% tab name="macOS" %}}
+
+```sh {id="terminal-prompt" class="command-line" data-prompt="$"}
+ffmpeg -i PATH_TO_GIF_OR_VID -vcodec libx264 -vf "format=yuv420p,scale=720:-1" -b:v 400k PATH_TO_GIF_OR_VID.mp4
+ffmpeg -i PATH_TO_GIF_OR_VID -c vp9 -b:v 0 -crf 41 my-animation.webm
+```
+
+{{% /tab %}}
+{{% tab name="Linux" %}}
+
+```sh {id="terminal-prompt" class="command-line" data-prompt="$"}
+ffmpeg -i PATH_TO_GIF_OR_VID -vcodec libx264 -vf "format=yuv420p,scale=720:-1" -b:v 400k PATH_TO_GIF_OR_VID.mp4
+ffmpeg -i PATH_TO_GIF_OR_VID -c:v libvpx-vp9 -b:v 0 -crf 41 my-animation.webm
+```
+
+{{% /tab %}}
+{{< /tabs >}}
+
+The first command:
+
+- uses the `libx264` codec
+- uses the `yuv420p` pixel format
+- scales the video to `720px` width
+- changes the bitrate to `300k` - you can change this value but check that the result is usable and reasonably small
```

Comment:
The command shown above uses `400k`.

The bash aliases below that wrap these commands use `300k`

Certainly the user can adjust this value, but ideally this doc use the same value throughout as a sort of baseline recommendation.